hera_qm/tests/test_xrfi.py

```diff
@@ -1285,7 +1298,19 @@ def test_threshold_wf_freq_broadcast():
     assert not uvf.flag_array[1].any()
 
 
-def test_threshold_wf_detrend():
+def test_threshold_wf_detrend(fake_waterfall):
+    uvm = fake_waterfall
+
+    # test with detrend
+    uvm.metric_array[50, :] += .5  # should get this
+    uvm.metric_array += .01 * np.arange(100).reshape((100, 1, 1))
+    uvf = xrfi.threshold_wf(uvm, nsig_f=5, nsig_t=5, detrend=True)
+    assert uvf.flag_array[50].all()
+    uvf = xrfi.threshold_wf(uvm, nsig_f=5, nsig_t=5, detrend=False)
+    assert not uvf.flag_array[50].all()
+
+
+def test_threshold_wf_detrend_no_check():
```

Comment:
Why do we test without the check? Does it increase coverage?

Reply:
I actually added this as a peace of mind so I knew that the check statements were hooked up correctly. I do not believe it has any affect on coverage.